### ISSUE TITLE
Why don't You return a database connection on connect method ?

### ISSUE THREAD

**Issue by mucyomiller** [ashokvishwakarma/dgraph-orm#12]
Why don't you return connection so that we can re-use it across the whole application
suppose we have node MVC application I don't see how I can connect to dgraph once with
```js
import dgraph from 'dgraph-orm';
import grpc from 'grpc';
dgraph.connect('HOST', 'PORT', grpc.credentials.createInsecure());
```
do I have to add above connection config in every and each file that talks to dgraph
why don't you return a connection so that I can connect to dgraph when am bootstrapping application and once connected use that connection across the whole application likes in dgraph-js 
eg:
```js
import * as dgraphjs from 'dgraph-js';
const clientStub = new dgraphjs.DgraphClientStub(
  process.env.DGRAPH_HOST_URI,
  grpc.credentials.createInsecure(),
);
export const db = new dgraphjs.DgraphClient(clientStub);
```
likes the above i can use that connection returned by DgraphClient for subsequent db queries
What do you think?

**Comment by ashokvishwakarma** [comment]
Note: Fixed in 1.2.1

```js
const connection = dgraph.connect(...);

// config: ConnectionConfig
connection.config

// client: DgraphClient
connection.client

// clientStub: DgraphClientStub
connection.clientStub

// dgraph - graph-js
connection.dgraph

// close: Function
connection.close();
```

Further to question "do I have to add above connection config in every and each file that talks to dgraph" the answer is NO as you should not use the connection object directly. Everything you need to run a query is available through models. Even you have a `query` and `mutate` method to run raw queries directly.

I don't see a use case for the same but I have exposed the connection object from the connect method by changing the function mentioned below

```js
/**
   * connect
   * 
   * @param config {ConnectionConfig}
   * 
   * @returns Connection
   */
  connect(config: ConnectionConfig): Connection  {
    this.connection = this._create_connection(config);
    return this.connection;
  }
```

**Comment by mucyomiller** [comment]
Okay Thanks, I was Using 
"dgraph-orm": "^1.1.3"
Let me try the newer version!
